File: _persist/temp/venda/CSharp/Cfop.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("CFOP")]
    public class Cfop : CollectionItem
    {
        [Campo("CD_CFOP", CampoTipo.tfKey)]
        public int Cd_Cfop { get; set; }
        [Campo("U_VERSION", CampoTipo.tfNul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.tfReq)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.tfReq)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("DS_CFOP", CampoTipo.tfReq)]
        public string Ds_Cfop { get; set; }
    }
}

File: _persist/temp/venda/Java/Cfop.java
package MORM.Java.Models;

import java.util.Date;
import MORM.Java.Classes;

@Tabela("CFOP")
public class Cfop extends CollectionItem
{
    @Campo("CD_CFOP", CampoTipo.tfKey)
    public Integer Cd_Cfop;
    @Campo("U_VERSION", CampoTipo.tfNul)
    public String U_Version;
    @Campo("CD_OPERADOR", CampoTipo.tfReq)
    public Integer Cd_Operador;
    @Campo("DT_CADASTRO", CampoTipo.tfReq)
    public DateTime Dt_Cadastro;
    @Campo("DS_CFOP", CampoTipo.tfReq)
    public String Ds_Cfop;
}